exercises/clock/examples/success-standard/src/Clock.hs

```diff
@@ -1,18 +1,16 @@
-module Clock (fromHourMin, clockHour, clockMin, toString) where
+module Clock (clockHour, clockMin, fromHourMin, toString, addDelta) where
```

Comment:
I think I see that the benefit of moving fromHourMin was to keep this in alphabetical order. In that case, `addDelta` should be the first element, should it not?